rclcpp/include/rclcpp/node.hpp

```diff
@@ -70,6 +70,8 @@
 #include "rclcpp/timer.hpp"
 #include "rclcpp/visibility_control.hpp"
 
+typedef std::function<void (rclcpp::CallbackGroup::SharedPtr)> CallbackGroupFunction;
```

Comment:
We should remove this, and instead rely on the version that lives in node_base.hpp.